### DISCUSSION TITLE
What is my Jekyll site missing ?

### DISCUSSION THREAD

**Discussion by moeenio** [community/community#22630] [Other Feature Feedback, Questions, & Ideas]
<p>Hi! I’m trying to make a Jekyll site, but the posts doesn’t appear on the front page.</p>
<p>My repo is <a href="https://github.com/locness3/locness3.github.io" rel="nofollow noopener">this</a> and it is on gh pages <a href="https://locness3.github.io" rel="nofollow noopener">here</a>.</p>
<p>Is my repo missing something ?</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>I created <a href="https://github.com/locness3/locness3.github.io/pull/4" rel="nofollow noopener">a pull request</a> that should fix things for you. You weren’t using a layout on your <code>index.html</code> file, which made Jekyll pass it by when it came to looking for liquid template strings.</p>
